Code/Engine/RendererFoundation/Device/Device.h

```diff
@@ -247,8 +241,8 @@ class EZ_RENDERERFOUNDATION_DLL ezGALDevice
 
   // Pipeline & Pass functions
 
-  virtual void BeginPipelinePlatform(const char* szName) = 0;
-  virtual void EndPipelinePlatform() = 0;
+  virtual void BeginPipelinePlatform(const char* szName, ezGALSwapChainHandle hSwapChain) = 0;
```

Comment:
Typically the SomethingPlatform functions already get a pointer to the corresponding GAL object and not the handle anymore.

Reply:
Fixed!